app/src/main/java/com/benefit/drivers/DatabaseDriver.java

```diff
@@ -22,23 +23,35 @@
 public class DatabaseDriver {
     private FirebaseFirestore db;
     private static final String TAG = "DatabaseDriver";
+    private FirebaseAuth auth;
```

Comment:
Let's create an AuthenticationDriver for the FirebaseAuth. the responsibility of this driver is working with the database, we should stick with the single responsibility principle.

Reply:
agree